### ISSUE TITLE
How to use NDI?

### ISSUE THREAD

**Issue by TimPietrusky** [vcync/modV#536]
**Is your feature request related to a problem? Please describe.**
Provide a guide on how to use NDI and link that in the tool tip inside of modV. 

**Describe the solution you'd like**
A guide on modv.vcync.gl on how to use NDI. 

**Describe alternatives you've considered**
Try to find out everything on my own.

**Comment by 2xAA** [comment]
We should probably simplify the process and add a group for NDI sources in the texture list.

**Comment by TimPietrusky** [comment]
@2xAA this would make a lot of sense.

**Comment by 2xAA** [comment]
I feel this issue is held up by #537 and my comment above should be its own issue.
